lib/p2p/Peer.ts

```diff
@@ -19,6 +19,9 @@ import NodeKey from '../nodekey/NodeKey';
 import Network from './Network';
 import Framer from './Framer';
 
+const dns = require('dns');
+const Traceroute = require('nodejs-traceroute');
```

Comment:
Aha, that would be why the Travis CI build is failing. Thanks!

Reply:
Yes, but in general we have some [Travis fixes pending](https://github.com/ExchangeUnion/xud/pull/1142). At the moment builds are **very** flaky - sorry for that.

Reply:
I added my package.json so that is all set. The dns module is a [core module](https://nodejs.org/api/dns.html), so I think it doesn't need to be listed in package.json